quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliCreateJvmApplicationIT.java

```diff
@@ -115,7 +112,7 @@ public void shouldAddAndRemoveExtensions() throws InterruptedException {
         assertTrue(result.isSuccessful(), SMALLRYE_HEALTH_EXTENSION + " was not uninstalled. Output: " + result.getOutput());
 
         // The health endpoint should be now gone
-        startAfter(app, Duration.ofSeconds(CMD_DELAY_SEC));
```

Comment:
There are unused constants left in this class: `CMD_DELAY_SEC` and `SMALLRYE_OPENAPI`. Plus the method `shouldAddAndRemoveExtensions` declares an `InterruptedException` which is never being thrown.

Maybe just do a revert of https://github.com/quarkus-qe/quarkus-test-suite/pull/361, that should take care of everything.